src/components/App/App.js

```diff
@@ -12,21 +12,19 @@ import Search from '../Search/Search'
 import Results from '../Results/Results'
 import Album from '../Album/Album'
 import Login from '../Login/Login';
+import UserContext from '../UserContext/UserContext'
 import mockUsers from '../MockData/mockusers';
 
 function App() {
   const [trendingData, setTrendingData] = useState()
-  const [currentUser, setCurrentUser] = useState(mockUsers[1]);
+  const [currentUser, setCurrentUser] = useState(false);
```

Comment:
no looks like this was broken previously and this is just revealing it lol - but still why is the default `false`?

Reply:
are we just gonna hide those tabs if not logged in? We should probably redirect to `/login` if a user manually types in those two paths.  Think, like a bookmark or something.  Or otherwise handle these errors.

Reply:
By setting currentUser to null, you clearly convey that there is no logged-in user